### PR TITLE
feat(Release): Get Data Other License, Linked Releases

### PR DESCRIPTION
**Pull request by tuannn2**
Create Release

- Get Data of Other License
- Get Data Linked Releases

LinkedRelease

### CODE REVIEW

File: src/components/ReleaseSummary/ReleaseSummary.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ import { VendorDialog } from '../sw360'
 import { GiCancel } from 'react-icons/gi'
 import ContributorsDialog from '../sw360/SearchContributors/ContributorsDialog'
 import ModeratorsDialog from '../sw360/SearchModerators/ModeratorsDialog'
+import OtherLicensesDiaglog from '../sw360/SearchOtherLicenses/OtherLicensesDialog'
```
Comment (heliocastro) [inline src/components/ReleaseSummary/ReleaseSummary.tsx:28]: Typo

File: src/components/ReleaseSummary/ReleaseSummary.tsx [hunk 2]
```diff
@@ -354,8 +369,15 @@ const ReleaseSummary = ({
                                 aria-describedby='Vendor'
                                 readOnly={true}
                                 name='otherLicenseIds'
+                                onClick={handleClickSearchOtherLicenses}
                                 value={otherLicensesId.fullName ?? ''}
                             />
+                            <OtherLicensesDiaglog
```
Comment (heliocastro) [inline src/components/ReleaseSummary/ReleaseSummary.tsx:375]: Typo ( Dialog x Diaglog )

File: src/components/sw360/SearchOtherLicenses/OtherLicensesDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+import SelectTableOtherLicenses from './SelectTableOtherLicenses'
+
+interface Props {
+    show?: boolean
+    setShow?: React.Dispatch<React.SetStateAction<boolean>>
+    session?: Session
+    selectLicenses?: LicensesType
+}
+
+const OtherLicensesDiaglog = ({ show, setShow, session, selectLicenses }: Props) => {
```
Comment (heliocastro) [inline src/components/sw360/SearchOtherLicenses/OtherLicensesDialog.tsx:32]: Type again

**Review by heliocastro — CHANGES_REQUESTED** [review]
Pleas fix the typo and will be ok

File: src/components/sw360/SearchOtherLicenses/OtherLicensesDialog.tsx (hunk 3 above)
Comment (tuannn2) [reply src/components/sw360/SearchOtherLicenses/OtherLicensesDialog.tsx]: fixed.

**Review by heliocastro — APPROVED** [review]

**Pull request merged**